Libplanet/Blockchain/BlockChain.cs

```diff
@@ -511,29 +511,24 @@ public Transaction<T> GetTransaction(TxId txId)
             Append(block, currentTime, evaluateActions: true, renderActions: true);
 
         /// <summary>
-        /// Adds <paramref name="transactions"/> to the pending list so that
-        /// a next <see cref="Block{T}"/> to be mined contains these
-        /// <paramref name="transactions"/>.
+        /// Adds <paramref name="transaction"/> to the pending list so that
+        /// a next <see cref="Block{T}"/> to be mined contains given
```

Comment:
```suggestion
        /// Adds a <paramref name="transaction"/> to the pending list so that
        /// a next <see cref="Block{T}"/> to be mined contains the given
```